packages/jest-cli/src/__tests__/__snapshots__/watch-test.js.snap

```diff
@@ -7,6 +7,7 @@ Watch Usage
  › Press o to only run tests related to changed files.
  › Press p to filter by a filename regex pattern.
  › Press t to filter by a test name regex pattern.
+ › Press r to open coverage report.
```

Comment:
@cpojer suggested one as per the conversations is `c` but it's been used already. I went with  `r` as in `reporting`. any suggestions?